lib/DDGC/DB/Result/InstantAnswer.pm

```diff
@@ -449,6 +450,12 @@ column deployment_state => {
     is_nullable => 1,
 };
 
+column blockgroup => {
+	date_type => 'varchar',
+	size => 20,
+	is_nullable => 1
```

Comment:
@zachthompson If you want to retrieve this in the repo/all JSON endpoint you need to add 'for_endpt => 1' here.
Not sure if you wanted that though!